1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -121,6 +122,7 @@ function generate_kubernetes_config() {
           <envVars/>
           <nodeSelector/>
           <remoteFs>/tmp</remoteFs>
+          <serviceAccount>${oc_serviceaccount_name}</serviceAccount>
```

Comment:
@ Is that function ever called? https://github.com/openshift/jenkins/search?utf8=%E2%9C%93&q=convert_is_to_slave&type=Code